src/features/comment/Comment.tsx

```diff
@@ -197,6 +198,8 @@ export default function Comment({
   // Comment from slice might be more up to date, e.g. edits
   const comment = commentById[commentView.comment.id] ?? commentView.comment;
 
+  const [selectTextModalIsOpen, setSelectTextModalIsOpen] = useState(false);
```

Comment:
Can we avoid putting this in Comment.tsx?

Perhaps we can resolve this with #169

Reply:
(this component is already doing way too much)

Reply:
Hmm. I suppose we can put this out and change its value with React context. How does that sound?